calendar-bundle/src/Resources/config/services.yml

```diff
@@ -7,6 +7,13 @@ services:
             calls:
                 - [setFramework, ['@contao.framework']]
 
+    contao_calendar.cron.generate_feeds:
+        class: Contao\CalendarBundle\Cron\GenerateFeedsCron
+        arguments:
+            - '@contao.framework'
+        tags:
+            - { name: contao.cronjob, interval: daily }
```

Comment:
Why are we not using a service tag like we do here?

https://github.com/contao/contao/blob/5b8c1d7beb54452b9a2451722b2742d1c026dca7/core-bundle/src/Cron/PurgePreviewLinksCron.php#L22

Reply:
Currently we sometimes tag our services via annotations and sometimes we tag them via YAML. There is no directive at the moment what is preferred.

As we do not use service auto registration and thus we need to define our services via YAML anyway I thought may be we should also just use YAML for tagging anyway?

Reply:
I prefer using annotations/attributes for those things that users are also using in their applications. Because then it's easier for people to understand what they need to do in their code.

Reply:
Not sure what you mean, in what way would you be using the cronjobs in your application for example?

Reply:
Maybe for importing things from an API, sending time based reminders, cleaning up custom tables…  All of that would be possible with a custom command and a real cronjob of course, but why not use our framework that makes it super easy to implement :slightly_frowning_face:.

Reply:
Oh, you mean in case someone is looking for examples when it comes to cronjobs etc.? idk., I would also make the argument that it might be useful to see the intervals of all cronjobs in one file (per bundle at least). Although I suppose that should be a command (#4640 😁).

Reply:
> Oh, you mean in case someone is looking for examples when it comes to cronjobs etc.?

Yes. And opinionated of course. :upside_down_face:

Reply:
But in that case we should expand that directive to hooks, callbacks and all event listeners, no?